lib/tasks/attempts.rake

```diff
@@ -0,0 +1,73 @@
+namespace :attempts do
+
+  AUTH_TOKEN = 'abc123'
+  PRIVATE_KEY_PATH = 'keys/attempts_api_private_key.key'
```

Comment:
in Ruby, constant defined inside a block like this go to the global/root namespace. I'd make these local variables (which are scoped to the block here)

```suggestion
  auth_token = 'abc123'
  private_key_path = 'keys/attempts_api_private_key.key'
```

Reply:
D'oh! I knew this in a former life. Good catch.